File: Aulas/Desenvolvimento Web II/2023-2/Aula_01/exercicio2.ts
//Faça um programa que receba três notas, calcule e mostre a média aritmética entre elas.
namespace exercicio2 {
    let nota1: number, nota2: number, nota3: number;

    nota1 = 8, nota2 = 4, nota3 = 9;
    const media: number = (nota1 + nota2 + nota3)/3

    if(media >= 7){
        console.log(`Aluno aprovado com média de: ${media}`)
    } else if (media >= 3){
        console.log(`Aluno está em recuperação com média de: ${media}`)
    } else {
        console.log(`Aluno reprovado com média de: ${media}`)
    }
}

File: Aulas/IoT/Codigos/Aula 01 - C/Exercicios/Exercicio2.c
//Faça um programa que receba três notas, calcule e mostre a média aritmética entre elas.

#include <stdio.h>

int main() {
    float nota1, nota2, nota3, media;

    printf("Digite as três notas: ");
    scanf("%f%f%f", &nota1, &nota2, &nota3);

    media = (nota1 + nota2 + nota3) / 3;

    printf("A média aritmética é: %.2f\n", media);

    return 0;
}